test/support/integration_case.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Livebook.IntegrationCase do
+  use ExUnit.CaseTemplate
+
+  using do
+    quote do
+      use LivebookWeb.ConnCase
+
+      @moduletag :integration
```

Comment:
We should be more specific and say "enterprise integration", since integration is very generic :)